res/css/structures/_RoomView.scss

```diff
@@ -130,7 +130,7 @@ limitations under the License.
 }
 
 .mx_RoomView_statusArea_expanded {
-    max-height: 100px;
+    max-height: 140px;
```

Comment:
Accommodate the longer content in the history import detected variation, especially when the error is showing and the text is wrapping to multiple lines.

<img width="678" alt="" src="https://user-images.githubusercontent.com/558581/170393433-136568e9-7331-42e0-840b-406d311dd291.png">

Reply:
This seems insufficient in languages with longer words.

Reply:
Agreed. The styles here are awkward and as noted in https://github.com/matrix-org/matrix-react-sdk/pull/8354#discussion_r882228796, we should clean them up in another PR.

For this PR, I can increase this more arbitrarily or remove if we're not worried about covering the whole screen?

Since we probably don't want to cover the whole screen, the separate PR iteration should probably make the content scroll at a certain point.